pkgs/games/osu-lazer/bypass-tamper-detection.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/osu.Game/OsuGameBase.cs b/osu.Game/OsuGameBase.cs
+index 98f60d52d..a27ce47ca 100644
+--- a/osu.Game/OsuGameBase.cs
++++ b/osu.Game/OsuGameBase.cs
+@@ -135,17 +135,7 @@ public OsuGameBase()
+         [BackgroundDependencyLoader]
+         private void load()
+         {
+-            try
+-            {
+-                using (var str = File.OpenRead(typeof(OsuGameBase).Assembly.Location))
+-                    VersionHash = str.ComputeMD5Hash();
+-            }
+-            catch
+-            {
+-                // special case for android builds, which can't read DLLs from a packed apk.
+-                // should eventually be handled in a better way.
+-                VersionHash = $"{Version}-{RuntimeInfo.OS}".ComputeMD5Hash();
+-            }
++            VersionHash = "253aa3a3a356a71295bf5b018cd4fda1";

```

Comment:
does this need to be touched along with other updates?

Reply:
From what I've tested no, it may just be that the versions were "close enough", but I don't think the version gets sent alongside this, the only place it seems to be used is here https://github.com/ppy/osu/blob/80496d8d7f436eb1fee0a87d35c0b5ccde450b7c/osu.Game/Screens/Multi/Play/TimeshiftPlayer.cs#L61

Reply:
okay